src/AgGrid.fs

```diff
@@ -41,57 +223,93 @@ type MenuItem =
     | BuiltIn of string
     | Custom of MenuItemDef
 
-[<Erase>]
-type IColumnDefProp<'row, 'value> = interface end
-let columnDefProp<'row, 'value> = unbox<IColumnDefProp<'row, 'value>>
-let columnDefProps<'row, 'value> = unbox<IColumnDefProp<'row, 'value>>
-
-[<Erase>]
-type IColumnDef<'row> = interface end
-
 type ColumnType = RightAligned | NumericColumn
 
 let openClosed = function | true -> "open" | false -> "closed"
 
 [<ReactComponent>]
-let CellRendererComponent<'value,'row> (render:'value -> 'row -> ReactElement, p) =
-    render p?value p?data
+let CellRendererComponent<'row, 'value> (render: (ICellRendererParams<'row, 'value>) -> ReactElement, p: ICellRendererParams<'row, 'value>) =

```

Comment:
I agree that this is probably a better interface, but it's a breaking change. We need to consider whether the trade-off is worth it.

Reply:
Given the style sheet is already a breaking change, maybe let's just bite the bullet and make a v2.

Would be good to add a migration guide in that case (can be minimal, a few sentences in the README or elsewhere might suffice).